### ISSUE TITLE
Ton force only in short ton force 

### ISSUE THREAD

**Issue by janheindejong** [hgrecco/pint#789]
First things first... absolutely LOVE this project. Great work! 

In my field of work, we do a lot of calculations in either meganewton, or metric ton-force. Of course I can use pint to convert between these units, by doing: 

```python
>>> my_force = 1 * ureg.t * ureg.gravity
>>> my_force.to(ureg.N)
<Quantity(9806.65, 'newton')>
```

There is also the option of using `ureg.ton_force`, which I feel is a bit cleaner, safer syntax. However, this unit refers to the short ton-force: 

```python
>>> my_force = 1 * ureg.ton_force
>>> my_force.to(ureg.N)
<Quantity(8896.443230521001, 'newton')>
```

Would it be possible to include the other definitions as well, i.e. as `ureg.metric_ton_force` and `ureg.long_ton_force` (and for consistency `ureg.short_ton_force`)?

**Comment by hgrecco** [comment]
I have no problem with including something else. But before doing it in the root of the registry, let's think if it would not be better to do it within a group or system 
See:
https://pint.readthedocs.io/en/0.9/systems.html

**Comment by janheindejong** [comment]
Although I'm still trying to understand how this works internally, it seems to make sense. What do you propose? For example: 

```python 
>>> (1 * ureg.sys.mks.ton_force).to('newton')
<Quantity(9806.65, 'newton')>
>>> (1 * ureg.sys.cgs.ton_force).to('newton')
<Quantity(9806.65, 'newton')>
>>> (1 * ureg.sys.imperial.ton_force).to('newton')
<Quantity(9964.02, 'newton')>
>>> (1 * ureg.sys.US.ton_force).to('newton')
<Quantity(8896.443230521001, 'newton')>
```

If we want to define them globally, I see a few options: 

|      Metric      |      Short      |      Long      | Description                                  |
|:----------------:|:---------------:|:--------------:|----------------------------------------------|
|    tonne_force   | short_ton_force | long_ton_force | verbose, according to wikipedia              |
|      t_force     | short_ton_force | long_ton_force | according to current definition of t in pint |
| metric_ton_force | short_ton_force | long_ton_force | verbose                                      |
| metric_ton_force |    ton_force    | long_ton_force | verbose, backward compatible                 |
|    tonne_force   |    ton_force    | long_ton_force | verbose, wikipedia, backward compatible      |
|      t_force     |    ton_force    | long_ton_force | current definition of t, backward compatible |

Any preference?

**Comment by hgrecco** [comment]
I think the second. Let's bring some people in. Any suggestion @jondoesntgit  and @cpascual ?

**Comment by hgrecco** [comment]
Closes due to #796
